### DISCUSSION TITLE
lxqt-openssh-askpass does not focus the input text field

### DISCUSSION THREAD

**Discussion by romildo** [lxqt/lxqt#2574] [General]
With version 2.0.0, `lxqt-openssh-askpass` does not focus the input text field automatically anymore. This is the place where one types the password. Now one has to press TAB twice to reach the text input field in order to type the password.

It would be very helpful to have the previous behavior restored.

**Comment by tsujan** [comment]
Nothing has changed in this regard. These are all changes from the previous version: https://github.com/lxqt/lxqt-openssh-askpass/compare/1.4.0...2.0.0. There's no change in focusing.

Check your WM.

**Reply by tsujan** [reply]
It's also possible that something has changed in Qt6. I vaguely remember a similar case in libfm-qt.

_If your WM focuses the dialog_ but the input isn't focused, please open an issue at https://github.com/lxqt/lxqt-openssh-askpass/issues.

**Comment by tsujan** [comment]
Closing this because you opened an issue: https://github.com/lxqt/lxqt-openssh-askpass/issues/96
